### ISSUE TITLE
ProfitLossFromOperatingActivities, expected content

### ISSUE THREAD

**Issue by mhoullier** [mhoullier/ESEF-block-tagging-hierarchy#8]
What should be the definition for this tag? Are all disclosures relative to components of the **profit (loss) from operating activities** expected to be covered by this tag as well?

For instance, should all sections tagged with the disclosure of **other operating income (expenses)** be tagged with this as well?

**Comment by Phamon** [comment]
No: this is a subtotal in the Income statement (as ProfitLoss). The tag should be used only if there is a specific disclosure referenced to profit (loss) from operating activities

**Comment by mhoullier** [comment]
I agree, this would be consistent with the XBRL-FR WG suggestion in #18
